testsuite/MDAnalysisTests/core/test_universe.py

```diff
@@ -216,6 +216,12 @@ def test_universe_topology_class_with_coords(self):
         assert_equal(u.trajectory.n_frames, u2.trajectory.n_frames)
         assert u2._topology is u._topology
 
+    def test_universe_from_smiles(self):
+        smi = "CN1C=NC2=C1C(=O)N(C(=O)N2C)C"
```

Comment:
The minimal dependency tests are failing here, you'll probably want an importorskip.